sql/catalyst/src/main/scala/org/apache/spark/sql/internal/StaticSQLConf.scala

```diff
@@ -54,6 +54,15 @@ object StaticSQLConf {
     .transform(_.toLowerCase(Locale.ROOT))
     .createWithDefault("global_temp")
 
+  val HIVE_METASTORE_PARTITION_PRUNING_INSET_THRESHOLD =
```

Comment:
Not a runtime config but a static config?

Reply:
Yes, for 2 reasons:
1. This parameter should be set according to your own Hive Metastore and does not need to be modified frequently.
2. All SQL configs in `HiveExternalCatalog` are static config, e.g.: `SCHEMA_STRING_LENGTH_THRESHOLD` and `DEBUG_MODE`.

Of course, we can make this parameter to runtime config if needed.

Reply:
hm, I see. I think users might lower the value on runtime just after they see the exception, so  IMO it is useful that users can update the value on runtime.